path_tolerance: 0.02
goal_tolerance: 0.0
test:
  joint_names:
  - joint_1
  - joint_2
  - joint_3
  - joint_4
  - joint_5
  - joint_6
  
home:
  positions:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0

esteso:
  positions:
  - [-1.0, 1.572, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  


M1:
  positions:
  - [0.0, 0.0, 1.572, 1.572, 1.470, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 


M2:
  positions:
  - [ 0.0, 0.3, 1.2708, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0

M3:
  positions:
  - [ 0.0, 1, 0.5708, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0

M4:
  positions:
  - [0.0, -1.47, 0.1, 1.572, 1.470, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

  
M5:
  positions:
  - [0.0, 0.0, -1.572, -1.572, -1.47, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 

G1:
  positions:
  - [0.0, 0.0, 1.0, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 

G2:
  positions:
  - [0.0, 0.5, 0.5, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 


M6:
  positions:
  - [0.0, -1.47, 2.0, 1.572, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0

Q1:
  positions:
  - [0.0, 0.0, 0.0, 0.0, -1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

Q2:
  positions:
  - [0.0, 0.0, 0.0, 0.0, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

Q3:
  positions:
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

Q4:
  positions:
  - [1.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

tmp:
  positions:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

Q5:
  positions:
  - [0.0, 0.2, 1.572, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

Q6:
  positions:
  - [0.0, 0.2, 1.572, -1.572, 1.572, -2.8]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 

Q7:
  positions:
  - [0.0, -0.2, -1.572, -1.572, -1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 

Q8:
  positions:
  - [0.0, 0.0, -1.572, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0   
  
Q9:
  positions:
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.5, -0.5, 1.472, 0.1, 1.472, 1.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, -1.572, 0.0, 0.0, 0.0]
  - [-0.5, -0.5, 1.472, 0.1, 1.472, 1.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, -1.572, 0.0, 0.0, 0.0]
  - [-0.3, -0.5, 1.472, 0.1, 1.472, 1.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.1, 0.0, -1.572, 0.0, 0.0, 0.0]
  - [0.3, -0.5, 1.472, 0.1, 1.472, 1.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 
  - 0.0 


Q10:
  positions:
  - [0.0, 0.4, 2.2, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

SPRING:
  positions:
  - [0.0, 0.0,    0.0, 0.0, 0.0, 0.0]
  - [0.1, 0.0,    1.5708, 1.5708, 0.0, 0.0]
  - [0.2, -1.5708, 1.5708, 0.0, 1.5708, 0.0]
  - [0.1, 0.0,   -1.5708, 0.0, 0.0, 1.5708]
  - [0.0, 0.0,    1.5708, 1.5708, 0.0, 0.0]
  - [-0.1, 0.0,    0.0, 0.0, 0.0, 1.5708]
  - [-0.2, 0.0,    -1.5708, 1.5708, 0.0, 0.0]
  - [0.0, 1.5708, -1.5708, 0.0, 1.5708, 0.0]
  - [0.1, 0.0,   1.5708, 0.0, 0.0, 1.5708]
  - [0.2, 0.0,    -1.5708, 1.5708, 0.0, 0.0]
  - [-0.2, 0.0,    0.0, 1.5708, 1.5708, 1.5708]
  - [0.0, 0.0,    0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
      
SPRING_JNT1:
  positions:
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  - [1.572,  0.0,   1.572, 0.0, 0.0, 0.0]
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  - [-1.572,  0.0,   1.572, 0.0, 0.0, 0.0]
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  - [1.572,  0.0,   1.572, 0.0, 0.0, 0.0]
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  - [-1.572,  0.0,   1.572, 0.0, 0.0, 0.0]
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  - [1.572,  0.0,   1.572, 0.0, 0.0, 0.0]
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  - [-1.572,  0.0,   1.572, 0.0, 0.0, 0.0]
  - [0.0,    0.0,   1.572, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  

SPRING_JNT2_ONESIDE:
  positions:
  - [0.0, 0.1, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 1.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.1, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 1.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.1, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
        
SPRING_JNT2:
  positions:
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 1.572, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, -1.572, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 1.572, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, -1.572, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 1.572, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, -1.572, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  

SPRING_JNT3:
  positions:
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572,  0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, -1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572,  0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, -1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572,  0.0, 0.0, 0.0]
  - [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [0.0, 0.0, -1.572, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  
SPRING_JNT4:
  positions:
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 1.572,  1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, -1.572, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 1.572,  1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, -1.572, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 1.572,  1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, -1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  
  
SPRING_JNT5:
  positions:
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0,  0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, -1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0,  0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, -1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0,  0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, -1.572, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  
SPRING_JNT6:
  positions:
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 1.572]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, -1.572]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 1.572]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, -1.572]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 1.572]
  - [0.0, 0.0, 1.572, 0.0, 0.0, 0.0]
  - [0.0, 0.0, 1.572, 0.0, 0.0, -1.572]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
  - 0.0  
esteso:
  positions:
  - [0.707, 1.5773545403248395, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

S1:
  positions:
  - [1.6009731459209693, 0.7, 2.2, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  
  
up:
  positions:
  - [0.707, 0.0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0  

P1:
  positions:
  - [1.6009731459209693, 0.8, 2.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  
P2:
  positions:
  - [1.6009731459209693, 0.0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  
P3:
  positions:
  - [1.6009731459209693, 0.25, 0.2, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  
P4:
  positions:
  - [ 0.00, 1.25, -1.5, 0.0, 1.57, 0.0]
  - [-0.5, 0.50, -1.75, 0.0, 1.57, 0.0]
  - [-1.0, 0.00, -0.75, 0.0, 1.57, 0.0]
  - [ 0.0, 1.25, -1.5, 0.0, 1.57, 0.0]
  - [ 0.0, 1.25,  1.0, 0.0, -1.57, 0.0]
  - [ 0.0, 1.25, -1.5, 0.0, 1.57, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  
  
T1:
  positions:
  - [1.6, 1.57, 1.3, 0.0, 0.0, 0.0]
  - [2.2, 1.57, 1.3, 0.0, 0.0, 0.0]
  - [-2.2, 1.57, 1.3, 0.0, 0.0, 0.0]
  - [1.6, 1.57, 1.3, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0

  
TEST1:
  positions:
  - [0, 1.57, 1.57, 0.0, 0.0, 0.0]
  - [0, 1.57, 0.0, 0.0, 0.0, 0.0]
  - [0, 1.57, 1.57, 0.0, 0.0, 0.0]
  - [1, 1.57, 1.57, 0.0, 0.0, 0.0]
  - [0, 1.57, 1.57, 0.0, 0.0, 0.0]
  - [0, 1.57, 0.0, 0.0, 0.0, 0.0]
  - [0, 1.57, 0.0, 1.0, 0.0, 0.0]
  - [0, 1.57, 0.0, 0.0, 0.0, 0.0]
  - [0, 1.57, 0.0, 0.0, 1.0, 0.0]
  - [0, 1.57, 0.0, 0.0, 0.0, 0.0]
  - [0, 1.57, 0.0, 0.0, 0.0, 1.0]
  - [0, 1.57, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0



Q5_loc_trj_jnt3:
  positions:
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.572, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
 

M4_loc_trj_jnt1:
  positions:
  - [ 0.0, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [-0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [-0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [-0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.0, -1.47, 0.1, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0 
  - 0.0 

Q5_loc_trj_jnt3:
  positions:
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.772, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.372, 1.572, 1.572, 0.0]
  - [0.0, 0.2, 1.572, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
 

M4_loc_trj_jnt1_2:
  positions:
  - [ 0.0, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [-0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [-0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [-0.3, -1.47, 0.1, 1.572, 1.572, 0.0]
  - [ 0.0, -1.47, 0.1, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0 
  - 0.0 

home_loc_trj_jnt1:
  positions:
  - [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [ 1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [-1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [ 1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [-1.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0


home_loc_trj_jnt2:
  positions:
  - [ 0.0,  0.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0,  1.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0, -1.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0,  1.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0, -1.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0,  0.0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0


home_loc_trj_jnt2_tmp:
  positions:
  - [ 0.0,  0.0, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0,  0.3, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0, -0.3, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0,  0.3, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0, -0.3, 0.0, 0.0, 0.0, 0.0]
  - [ 0.0,  0.0, 0.0, 0.0, 0.0, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0


  
M1_loc_trj_jnt1:
  positions:
  - [ 0.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [-1.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  - [ 0.0, 0.0, 1.5708, 1.572, 1.572, 0.0]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0

pack:
  positions:
  - [-1.574343656368829, 0.35128160020311733, -1.9357878792852463, -0.00019174759836414697, -1.5692623450121792, -0.0023968449795518368]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0

Stage1:
  positions:
  - [2.4286, 0.3448, 0.9093, 0.115, 0.7712, 2.5487]
  - [-2.1964, -1.5945, -0.1865, 2.5056, 0.2215, 2.5434]
  - [-1.6632, 0.6756, -1.5272, 1.754, 1.2175, -1.7816]
  - [2.2618, 0.9198, -0.5923, 2.7032, -0.4543, 1.6512]
  - [-1.0632, 1.7277, -1.0015, 1.0805, -0.6021, 0.9483]
  - [-1.0771, -0.8775, -1.182, 2.7069, 0.7922, -2.6767]
  - [2.3979, 1.2443, -2.1729, -0.4479, 0.9775, 0.6348]
  - [2.117, -1.6742, 1.7021, -1.9926, -1.4944, 0.6347]
  - [-1.908, 1.5971, -0.1004, -1.6362, -0.448, -1.1636]
  - [1.817, 0.8134, -0.9606, -1.889, -0.7097, -2.0373]
  - [0.9545, -1.7116, 0.6423, 2.586, -0.8583, -0.866]
  - [0.4749, -1.4356, -0.6099, -0.289, -0.8068, -0.6573]
  - [-2.4501, -0.4176, -0.7907, 0.112, 0.2854, 1.0874]
  - [1.6749, -0.2237, -0.3822, -2.1304, 0.3149, 1.1031]
  - [1.1872, 0.1626, -1.7342, -2.3462, 1.4707, -0.5719]
  - [-0.1816, -1.654, 1.331, 1.6288, 0.191, -0.6793]
  - [0.0231, -0.579, 1.5662, 2.3598, -0.0536, -0.4786]
  - [0.7037, 0.7711, -0.4727, -0.9152, 1.0611, 1.2001]
  - [-2.0932, 0.2776, 1.0626, 2.3598, 1.3621, 1.0364]
  - [2.5818, 0.5148, -1.3056, -2.1346, 0.7068, -1.5536]
  velocities:
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  - [0.5, 0.5, 0.5, 0.5, 0.5, 0.5]
  time_from_start:
  - 0.0
  - 9.5
  - 18.4
  - 26.5
  - 33.4
  - 40.9
  - 48.1
  - 56.1
  - 64.4
  - 72.1
  - 81.3
  - 87.3
  - 93.4
  - 101.9
  - 105.5
  - 113.7
  - 116.1
  - 122.9
  - 129.7
  - 139.3